query-engine/prisma-models/Cargo.toml

```diff
@@ -9,14 +9,22 @@ itertools = "0.10"
 prisma-value = { path = "../../libs/prisma-value" }
 bigdecimal = "0.3"
 thiserror = "1.0"
-
+getrandom = { version = "0.2" }
 uuid = { workspace = true, optional = true }
-cuid = { version = "1.2", optional = true }
+cuid = { git = "https://github.com/prisma/cuid-rust", branch = "jkomyno/wasm32", optional = true }
 nanoid = { version = "0.4.0", optional = true }
 chrono = { version = "0.4.6", features = ["serde"] }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies.getrandom]
+version = "0.2"
+features = ["js"]
+
 [features]
-# Support for generating default UUID, CUID, nanoid and datetime values. This
-# implies random number generation works, so it won't compile on targets like
-# wasm32.
-default_generators = ["uuid/v4", "cuid", "nanoid"]
+default = []
+
+# Support for generating default UUID, CUID, nanoid and datetime values.
+default_generators = ["cuid", "uuid", "nanoid"]
+
+cuid = ["dep:cuid"]
+nanoid = ["dep:nanoid"]
+uuid = ["uuid/v4"]
```

Comment:
To the reviewer: feel free to revert this change (maintaining the comment) with: 

```suggestion
# Support for generating default UUID, CUID, nanoid and datetime values.
default_generators = ["uuid/v4", "cuid", "nanoid"]
```